templates/rails/files/.platform.app.yaml

```diff
@@ -42,8 +42,19 @@ mounts:
 hooks:
     # The build hook turns what is in source control into the deployable application.
     build: |
-        gem install bundler:2.0.2
-        bundle install --without development test
+        gem install --no-document bundler:2.1.4
+        bundle config set without 'development test'
+        bundle config deployment true
+        bundle config cache_all true
+        bundle config clean true
+        echo "Installing gems"
+        # when cache directory exists apply it to app so bundle use it
```

Comment:
```suggestion
        # Rails Gems don't have a reusable cache, but can benefit from having the
        # previous build available to save time. We therefore copy the downloaded gems
        # to the Platform.sh cache directory for safe keeping, then restore from there on
        # the next build.  That allows bundler to skip downloading code it doesn't need to.
```

Reply:
Rails Gems do have reusable cache. This currently does not keep only cache directory but the whole bundler directory to avoid recompiling compiled extensions, I've pushed a modified version of your suggestion feel free to improve thanks